### ISSUE TITLE
[C++] alt + q not working for constructors in derived class

### ISSUE THREAD

**Issue by jasjuang** [20Tauri/DoxyDoxygen#123]
Here is a minimal example to reproduce it

```
class Derived : public Base
{
 public:

  Derived();

};
```

alt+Q when cursor is on the constructor yields nothing. However, if we take out the inheritance

```
class Derived
{
 public:

  Derived();

};
```

everything works like usual. Would be nice to see this fixed!

**Comment by 20Tauri** [comment]
Thanks for reporting,

"Derived" wasn't detected a a class name. The fix will be deployed next week on evolution branch

**Comment by 20Tauri** [comment]
Fixed in [0.68.3](https://github.com/20Tauri/DoxyDoxygen/releases/tag/0.68.3)
Available from now in evolution branch.
